cmd/thanos/tools_bucket.go

```diff
@@ -313,8 +326,15 @@ func registerBucketInspect(app extkingpin.AppClause, objStoreConfig *extflag.Pat
 		for _, meta := range metas {
 			blockMetas = append(blockMetas, meta)
 		}
-
-		return printTable(blockMetas, selectorLabels, *sortBy)
+		if opType == TABLE {
+			return printBlockData(blockMetas, selectorLabels, *sortBy, printTable)
+		} else if opType == TSV {
+			return printBlockData(blockMetas, selectorLabels, *sortBy, printTSV)
+		} else if opType == CSV {
+			return printBlockData(blockMetas, selectorLabels, *sortBy, printCSV)
+		} else {
+			return fmt.Errorf("Invalid output type %s.", *output)
```

Comment:
Typically error messages start with a lower letter because they look properly then in case they get chained e.g. `doing: Creating: linking: ...` vs. `doing: creating: linking: ...`. In fact, this case would be handled for you if you'd use enums as recommended previously

Reply:
got it thanks. will keep in mind for future contributions